### PR TITLE
Speed up casting by not killing apps known to be nonblocking

### PR DESCRIPTION
**Pull request by theychx**
Greetings! Just a little improvement to make catt faster

### CODE REVIEW

File: catt/controllers.py [hunk 1]
```diff
@@ -66,8 +66,12 @@ def __init__(self):
         time.sleep(0.2)
 
     def play_media(self, url, content_type="video/mp4"):
-        self.kill()
-        time.sleep(5)
+        NON_BLOCKING_APPS = ['Backdrop', 'Default Media Receiver',
+                             'BubbleUPnP']
+        if self.cast.app_display_name not in NON_BLOCKING_APPS:
+            self.kill()
+            while self.cast.app_display_name != 'Backdrop':
```
Comment (skorokithakis) [inline catt/controllers.py]: You're sure this updates every call, right?

**Comment by skorokithakis** [comment]
Hello! Can you explain the rationale behind this a bit, and also how it works?

**Comment by theychx** [comment]
> You're sure this updates every call, right?
Yep, tested it with serveral apps, it always gets the id right.

The point is that only a genuine "Custom Receiver" app (such as YouTube) might  exhibit the blocking behaviour. Theres no need to kill the currently running app and wait 5 seconds, if this app will let go of control voluntarily.
In the case of a  "Custom Receiver" app running, I changed the code so instead of just waiting 5 seconds, we poll for "Backdrop" to appear, and thus save some time here as well.

PS: I added BubbleUpnp to the list of nonblocking apps because it is an iteration of the "Styled Media Receiver" which I believe behaves just as the "Default Media Receiver"

**Comment by skorokithakis** [comment]
Fantastic, thank you very much for the contribution!

**Pull request merged**